packages/valory/skills/registration_abci/behaviours.py

```diff
@@ -72,7 +72,7 @@ def async_act(self) -> Generator:
             initialisation = (
                 json.dumps(self.synchronized_data.db.setup_data, sort_keys=True)
                 if self.synchronized_data.db.setup_data != {}
-                else None
+                else ""
```

Comment:
See previous comment. Suggest to make empty dict serialised

Reply:
ok, so effectively `json.dumps({}, sort_keys=True)` when `setup_data` is an empty dict.